### PR TITLE
Add documentation for admin 

### PR DESCRIPTION
**Pull request by clytemnestra**
### Subject

 Added documentation about how to remove Admin, maybe add more info about it in the future. We need an explanation about the admin interface and how to remove it, if people don't want additional fields in their admin dashboard.

### To do

- [ ] wait for opinions
- [ ] squash commits

### CODE REVIEW

File: Resources/doc/reference/admin.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Admin
+=======
+
+The bundle comes with an Admin interface for managing your contexts, tags, collections and categories.
+
```
Comment (Soullivaneuh) [inline Resources/doc/reference/admin.rst]: Remove one line.

File: Resources/doc/reference/admin.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
+Admin
+=======
```
Comment (Soullivaneuh) [inline Resources/doc/reference/admin.rst]: Should be same length as the title.

**Comment by Soullivaneuh** [comment]
Please change you commit title.

"Create admin.rst" means nothing clear.

**Comment by core23** [comment]
Maybe we should move the docs to the AdminBundle instead of copy & paste it for every bundle

**Comment by clytemnestra** [comment]
The thing is sometimes the admin services are required and you can't remove them from the container without having to override the places where they are used. And that is specific to each bundle.

File: Resources/doc/reference/admin.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+              $tags['sonata.admin'][0]['show_in_dashboard'] = false;
+              $definition->setTags($tags);
+  
+          }
+      }
+  }
+
+
+
+Note that this will keep the services and the routes in the container, it will just not show links in the dashboard, anymore.
```
Comment (greg0ire) [inline Resources/doc/reference/admin.rst]: "dashboard anymore"

File: Resources/doc/reference/admin.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+If you don't want the admin routes to be accessible, either, then you will have to remove the service definitions from the container.
```
Comment (greg0ire) [inline Resources/doc/reference/admin.rst]: remove extra comma

**Comment by clytemnestra** [comment]
Squashed and changed

**Comment by core23** [comment]
I don't like the idea to create a whole class in the docs which removes the automapping to the admin. Maybe we should find another way to include this feature in the bundle. Maybe via a config key?

Same as https://github.com/sonata-project/SonataMediaBundle/pull/1048

**Comment by clytemnestra** [comment]
That could work, too. But then you'd have to add a conf. key for every admin service, in case one wants to remove just some of them.

**Comment by clytemnestra** [comment]
Well I guess if you'd want one removed you would want them all and there's always this alternative in case you wanted only a few removed.

**Comment by core23** [comment]
IMO, we don't need any special information for this rare case.

You could also configure your admins in the [configuration](https://sonata-project.org/bundles/admin/3-x/doc/reference/dashboard.html#using-the-config-yml).

**Pull request closed without merging**